opm/output/eclipse/RestartIO.cpp

```diff
@@ -205,6 +223,27 @@ namespace {
         rstFile.write("ZGRP", groupData.getZGroup());
     }
 
+    void writeGroupLGR(int                           sim_step,
+                       const UnitSystem&             units,
+                       const Schedule&               schedule,
+                       const Opm::SummaryState&      sumState,
+                       const std::vector<int>&       ih,
+                       EclIO::OutputStream::Restart& rstFile,
+                       const std::string&            lgr_tag)
+    {
+        // write IGRP to restart file
+        // write IGRP to restart file
```

Comment:
This (duplicated) comment doesn't add any context as everyone can see that that's what we're doing.  Please remove this.